examples/structured_2d_dgsem/elixir_eulerpolytropic_coupled.jl

```diff
@@ -0,0 +1,139 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Couple polytropic Euler systems.
+#
+# In a rectangular domain we solve two different sets of equations for the
+# left and the right half. Those are the isothermal equations (left)
+# and polytropic equations (right).
+# The coupling hapens on two interfaces. One is located at the center of the
+# domain such that the right half is coupled through its left boundary
+# and the left half is coupled through its right boundary. The other coupling
+# makes sure that the domain is periodic. Here the right boundary of the right
+# domain is coupled to the left boundary of the left domain.
+# The vertical boundaries are simply periodic.
+# As test case we send a linear wave through the domain and observe a change
+# in the dispersion relation when the wave enters the isothermal domain.
+
+###############################################################################
+# define the initial conditions
+
+function initial_condition_wave_isothermal(x, t, equations::PolytropicEulerEquations2D)
+    equations.gamma = 1.0
+    equations.kappa = 1.0
```

Comment:
These two lines should be removed, I think.

Reply:
Of course. It's removed.